lua/mason-registry/luaformatter/init.lua

```diff
@@ -0,0 +1,23 @@
+local Pkg = require "mason-core.package"
+local git = require "mason-core.managers.git"
+local github = require "mason-core.managers.github"
+local platform = require "mason-core.platform"
+local Optional = require "mason-core.optional"
+
+return Pkg.new {
+    name = "luaformatter",
+    desc = [[Code formatter for Lua]],
+    homepage = "https://github.com/Koihik/LuaFormatter",
+    languages = { Pkg.Lang.Lua },
+    categories = { Pkg.Cat.Formatter },
+    ---@async
+    ---@param ctx InstallContext
+    install = function(ctx)
```

Comment:
It seems they distribute via luarocks, which they also recommend. I think we should use that method to install (there's already a luarocks manager, might need to be slightly extended though)

Reply:
i actually tried it implementing with luarocks but i couldn't get it working. they also seems didn't update dev branch in a while. so, i thought going with c(make) would be good choice.